.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-*'
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  release:
+    runs-on: windows-2019 # Code signing requirement https://github.com/NuGet/Home/issues/7939
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1.8.2
+        with:
+          dotnet-version: 5.0.x
+      - name: Build
+        run: dotnet build src --configuration Release
+      - name: Install NuGetKeyVaultSignTool
+        run: dotnet tool install --global NuGetKeyVaultSignTool
+      - name: Sign NuGet Packages
+        run: |
+          NuGetKeyVaultSignTool sign nugets\*.nupkg `
+          --file-digest sha256 `
+          --timestamp-rfc3161 http://timestamp.digicert.com `
+          --timestamp-digest sha256 `
+          --azure-key-vault-url https://particularcodesigning.vault.azure.net `
+          --azure-key-vault-client-id ${{ secrets.AZURE_KEY_VAULT_CLIENT_ID }} `
+          --azure-key-vault-tenant-id ${{ secrets.AZURE_KEY_VAULT_TENANT_ID }} `
+          --azure-key-vault-client-secret ${{ secrets.AZURE_KEY_VAULT_CLIENT_SECRET }} `
+          --azure-key-vault-certificate ${{ secrets.AZURE_KEY_VAULT_CERTIFICATE_NAME }}
+        shell: pwsh
+      - name: Publish artifacts
+        uses: actions/upload-artifact@v2.2.2
+        with:
+          name: nugets
+          path: nugets/*
+          retention-days: 1
+      - name: push
+        env:
+          API_KEY: ${{ secrets.MYGET_PUBLISH_API_KEY }}
+        if: (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release-') || startsWith(github.ref, 'refs/tags/')) && env.API_KEY != ''
```

Comment:
Where did this condition come from? That does not look correct. This workflow only triggers on a tag push, so it shouldn't need to be checking any of that.